scripts.js

```diff
@@ -230,11 +230,7 @@ function details(institution) {
             studentMajor = document.createElement('p');
         studentPhoto.src = 'portraits/' + currentYear + '/' + student.name + '.jpg';
         imageExists(studentPhoto.src, function(exists) {
-            if(exists == true)
-            {
-                studentPhoto.src = 'portraits/' + currentYear + '/' + student.name + '.jpg';
-            }
-            else {
+            if(!exists) {
```

Comment:
`if (`

Reply:
The default (name + jpg) is already set above because the function needs the file path to see if it exists -- if it does nothing gets changed, so I was thinking setting it to the blank dude picture if it *doesn't* (!exists) exist seemed the most concise. it could also be 
`studentPhoto.src = exists ? studentPhoto.src : 'portraits/blankHead.png';` (i think that's right at least hadn't heard of ternary statements) but that also seemed a lil repetitive because you're not changing anything if it's true.

whatever u and andrew think is best -  if it's not abundantly clear i dont actually know how to code

Reply:
on further reflection, i have learned that you meant just put a space before the if

my aplogies